lib/network.js

```diff
@@ -96,7 +97,29 @@ Network.prototype.start = function (callback) {
             self.socket.setBroadcast(true);
 
             //TODO: get the default broadcast address from os.networkInterfaces() (not currently returned)
-            self.destination = [self.broadcast || "255.255.255.255"];
+
+            if (self.broadcast) {
```

Comment:
@wankdanker this handles if the user DOES manually define broadcast as an option, to use what they supplied. Also expands the API to support passing an array instead of a single value.

The new code below only activates if the user did not manually declare broadcast.